architecture/core_concepts/templates.adoc

```diff
@@ -87,12 +87,14 @@ application.
 
 Templates can be processed from a definition in a file or
 from an existing OpenShift API object.  Cluster administrators can
-ifdef::openshift-enterprise,openshift-origin[]
-link:../../install_config/install/first_steps.html#creating-instantapp-templates[define
-standard templates in the API]
+ifdef::openshift-origin[]
+link:../../install_config/install/first_steps.html#creating-instantapp-templates[define standard templates in the API]
+endif::[]
+ifdef::openshift-enterprise[]
+link:../../install_config/default_imagestreams_templates.html#creating-instantapp-templates[define standard templates in the API]
```

Comment:
why do we have different locations for the same material between origin and OSE?

I also see that even the if-def:origin content in the "default_imagestreams_templates" file talks about this stuff, so why not link there?  (same for all the other cases below)